src/Form/PosterType.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\Poster;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\FileType;
+use Symfony\Component\Validator\Constraints\File;
+
+class PosterType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('file_name', TextType::class)
+            ->add('poster_img', FileType::class, [
+                'label' => 'Image (jpg/jpeg/png file)',
+                'mapped' => false,
+                'required' => true,
+                'constraints' => [
+                    new File([
+                        'maxSize' => '1024k',
+                        'mimeTypes' => [
+                            'image/jpeg',
+                            'image/jpg',
```

Comment:
I think the image/jpg MIME type doesn't exist

Reply:
Ok, I took it out. Image/jpeg also check jpg extension's...